src/SprykerSdk/Zed/AppSdk/Business/AsyncApi/Builder/AsyncApiBuilder.php

```diff
@@ -118,28 +128,8 @@ protected function addComponentMessage(array $asyncApi, string $messageName, Asy
     protected function addComponentSchemaMessage(array $asyncApi, string $messageName, AsyncApiMessageTransfer $asyncApiMessageTransfer): array
     {
         $messageAttributes = $this->getMessageAttributes($asyncApiMessageTransfer);
```

Comment:
Why do I see the same changes in two PRs? https://github.com/spryker-sdk/app-sdk/pull/8/files has the same changes. 

You need to be aware of this.